acados/dense_qp/dense_qp_common.c

```diff
@@ -85,6 +89,18 @@ dense_qp_dims *dense_qp_dims_assign(void *raw_memory)
     return dims;
 }
 
+
+void dense_qp_dims_set(void *config_, void *dims_, const char *field, const int* value)
+{
+    // wrap hpipm function
+    char field_copy[MAX_STR_LEN];
```

Comment:
why do you need field_copy?

Reply:
because the hpipm setter that is wrapped by this function takes `char *` instead of `const char *`.
If the function is called from C++, one will get compiler warnings otherwise. See, https://github.com/giaf/hpipm/pull/54

Reply:
it can not be simply cast?